src/component/book.js

```diff
@@ -1,23 +1,31 @@
+import React, { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/books';
+import { removeBook, fetchBooks } from '../redux/books/books';
 import './books.css';
 
 const Book = () => {
-  const books = useSelector((state) => state.booksReducer);
+  const books = useSelector((state) => state.bookStore);
   const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchBooks());
+  }, []);
   return (books.map((book) => (
     <div key={book.id} className="card">
       <div className="main-area">
         <span>Action</span>
-        <h1>{book.title}</h1>
-        <p>{book.author}</p>
+        <p>{book.category}</p>
+        <h4>{book.title}</h4>
+        <h6>{book.author}</h6>
         <ul>
           <button type="button">Comments</button>
           <button
             type="button"
+            id={book.id}
+            value="Remove"
             onClick={(e) => {
               e.preventDefault();
-              dispatch(removeBook(book.id));
+              dispatch(removeBook(e.target.id));
+              dispatch(fetchBooks());
             }}
           >
```

Comment:
- [ ] The name of this component is `book.js` while the content is list of books. I suggest the appropriate name would be `books.js` or `bookList.js`.
- [ OPTIONAL ] It would be great, if you can decomose this componet further into a Book.js child component. For example:
```
 <ul>
   (books.map((book) => {
     <Book book={book} />
   })
</ul>
```

```
Inside Book.js component

return (
 <li key=book.id >
   ....
 </li>
)
```